src/main/java/carpet/script/utils/ShapeDispatcher.java

```diff
@@ -625,6 +629,69 @@ public static int particleMesh(List<ServerPlayer> playerList, ParticleOptions pa
         }
     }
 
+    public static class Polyface extends ExpiringShape
+    {
+        @Override
+        public long calcKey(){
+            long hash = super.calcKey();
+            hash ^= 2;                     hash *= 1099511628211L;
+            hash ^= Byte.hashCode(mode);   hash *= 1099511628211L;
+            hash ^= going.hashCode();      hash *= 1099511628211L;
+            for (Vec3 i :vertex_list){
+                hash ^= vec3dhash(i);
+                hash *= 1099511628211L;
+            }
+            hash ^= vertex_list.size();    hash *= 1099511628211L;
+            hash ^=(vertex_list.size()+37)*(vertex_list.size()+3721)*4224696537L+30692*vertex_list.size();
```

Comment:
why this?

Reply:
removed.
some excitement came on as I was writing.🙃🤦‍♂️